Light Vox Engine/ECS/EntityComponents.h

```diff
@@ -8,13 +8,15 @@
 
 namespace EntityComponents
 {
-    struct Transform
+    struct TransformMatrix
     {
         glm::mat4 transformMatrix;
```

Comment:
Why not do a define based on platform? Whats the point of wrapping it in a struct? 

Maybe a `using` statement could be better?

Reply:
This should mirror whatever graphics has -- if we end up with a material system as well, this struct should also contain an index or something for that.